### PR TITLE
PREAPPS-4344 : Re-skin 'External User Registeration' page

### PR DESCRIPTION
**Pull request by kvc2983**

### CODE REVIEW

File: WebRoot/public/extuserprov.jsp [hunk 1]
```diff
@@ -55,19 +55,18 @@
 </head>
 <body>
 
-<div class="LoginScreen">
-		<div class="center">
-			<div class="contentBox">
+<div id="modifiedLogin" class="LoginScreen">
```
Comment (silentsakky) [inline WebRoot/public/extuserprov.jsp:58]: is it necessary to have id?

Comment (kvc2983) [reply WebRoot/public/extuserprov.jsp]: Removed.

File: WebRoot/public/extuserprov.jsp [hunk 2]
```diff
@@ -77,37 +76,44 @@
...
+						</tr>
+						<tr>
+							<td><input id="displayname" class="zLoginField" name="displayname" type="text" value="${fn:escapeXml(param.displayname)}" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/></td>
+						</tr>
+						<tr>
+							<td><label class="zLoginFieldLabel" for="password"><fmt:message key="password"/></label></td>
+						</tr>
+						<tr>
+							<td><input id="password" class="zLoginField" name="password" type="password" value="${fn:escapeXml(param.password)}" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/></td>
+						</tr>
```
Comment (auminpatel) [inline WebRoot/public/extuserprov.jsp:103]: for type="password" use ` class=zLoginFieldInput ` have a look at the login page , try to make it consistent

File: WebRoot/public/extuserprov.jsp [hunk 3]
```diff
@@ -77,37 +76,44 @@
...
+							<td><input id="password2" class="zLoginField" name="password2" type="password" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/></td>
+						</tr>
+						<tr>
+							<td style="text-align:right; padding-bottom: 20px;">
+								<input type="submit" class="loginButton" value="<fmt:message key="register"/>" style="float:left;"/>
+						</tr>
+					</c:otherwise>
+					</c:choose>
+				</div>
 			</table>
```
Comment (auminpatel) [inline WebRoot/public/extuserprov.jsp:118]: Just a thougth , there needs to be a bit more padding at the bottom , acccording to screenshot
have a look at https://apps-development.synacor.tk/public/PasswordRecovery.jsp

Comment (kvc2983) [reply WebRoot/public/extuserprov.jsp:118]: Actually Joshua Suggested me to give 20px padding so I need to honor his suggestion.

File: WebRoot/public/extuserprov.jsp [hunk 4]
```diff
@@ -77,37 +76,44 @@
...
+						</tr>
+						<tr id="confirmPassword" style="display:none">
+							<td><label class="zLoginFieldLabel" for="password2"><fmt:message key="confirm"/></label></td>
+						</tr>
+						<tr>
+							<td><input id="password2" class="zLoginField" name="password2" type="password" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/></td>
+						</tr>
+						<tr>
+							<td style="text-align:right; padding-bottom: 20px;">
+								<input type="submit" class="loginButton" value="<fmt:message key="register"/>" style="float:left;"/>
```
Comment (auminpatel) [inline WebRoot/public/extuserprov.jsp:112]: avoid inline styling

File: WebRoot/public/extuserprov.jsp [hunk 5]
```diff
@@ -77,37 +76,44 @@
...
+							<td><input id="password" class="zLoginField" name="password" type="password" value="${fn:escapeXml(param.password)}" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/></td>
+						</tr>
+						<tr id="confirmPassword" style="display:none">
+							<td><label class="zLoginFieldLabel" for="password2"><fmt:message key="confirm"/></label></td>
+						</tr>
+						<tr>
+							<td><input id="password2" class="zLoginField" name="password2" type="password" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/></td>
+						</tr>
+						<tr>
+							<td style="text-align:right; padding-bottom: 20px;">
```
Comment (auminpatel) [inline WebRoot/public/extuserprov.jsp:111]: avoid inline styling

File: WebRoot/public/extuserprov.jsp (hunk 2 above)
Comment (kvc2983) [reply WebRoot/public/extuserprov.jsp:103]: Done

File: WebRoot/public/extuserprov.jsp (hunk 5 above)
Comment (kvc2983) [reply WebRoot/public/extuserprov.jsp]: done

**Pull request merged**